lego-webapp/components/JoblistingItem/index.tsx

```diff
@@ -64,20 +64,21 @@ const JoblistingItem = ({ joblisting }: JobListingItemProps) => (
           )}
         </Flex>
       </Flex>
-      <Flex
-        alignItems="center"
-        gap="var(--spacing-sm)"
-        className={styles.deadline}
-      >
-        <Icon iconNode={<CalendarClock />} size={16} />
-        {joblisting.rollingRecruitment ? (
-          'Snarest'
-        ) : (
-          <Time
-            time={joblisting.deadline}
-            format={`ll ${moment(joblisting.deadline).format('HH:mm') !== '23:59' ? 'HH:mm' : ''}`}
-          />
-        )}
+      <Flex className={styles.infoLeft}>
+        <div className={styles.pin}>
+          {joblisting.isPinned && <Icon iconNode={<Pin />} size={16} />}
+        </div>
```

Comment:
```suggestion
        {joblisting.isPinned && (
          <div className={styles.pin}>
            <Icon iconNode={<Pin />} size={16} />}
          </div>
        )}
```

Dont need to render the div if pin is not there?